json_logic/__init__.py

```diff
@@ -121,14 +121,23 @@ def get_var(data, var_name, not_found=None):
         return data
 
 
-def get_date(date_str, *args):
+def get_date(value, *args):
+    if isinstance(value, date):
```

Comment:
I added this, because currently we have logic rules that look like this:
```json
{
    "<": [
        {"date": {"var": "dateOfBirth"}},
        {"date": "2021-01-01"},
    ]
}
```
But the data is converted to python type, which means that `{"var": "dateOfBirth"}` is already a date. 

To avoid modifying all rules, I added this workaround until we sort out the data typing.